src/distilabel/pipeline/local.py

```diff
@@ -658,10 +659,28 @@ def _stop(self) -> None:
 
         with _STOP_CALLED_LOCK:
             if _STOP_CALLED:
-                self._logger.warning(
-                    "🛑 Stop has already been called. Ignoring subsequent calls and waiting"
-                    " for the pipeline to finish..."
-                )
+                global _STOP_CALLS
+                _STOP_CALLS += 1
+                if _STOP_CALLS == 1:
+                    self._logger.warning(
+                        "🛑 Stop has already been called. Ignoring subsequent calls and waiting"
+                        " for the pipeline to finish..."
+                    )
+                elif _STOP_CALLS == 2:
+                    self._logger.warning(
+                        "🛑 Press again to force interruption of the pipeline."
+                    )
+                else:
+                    self._logger.warning("🛑 Forcing pipeline interruption.")
+                    import gc
+                    import sys
+
+                    if manager:
+                        manager.shutdown()
+                    gc.collect()
+
+                    sys.exit(1)
```

Comment:
I think we can force the stop at second CTRL+C and updating the first log message to include something like: "Press again to force pipeline stop" should be enough.

Reply:
Done